### PR TITLE
 Common behaviour for lists with items that have address part 2

### PR DESCRIPTION
**Pull request by karliss**
**Detailed description**

Part 2 for #1700  and #923

**Test plan (required)**

* Test Resource widget
    * double click seek
    * show in hexwidget
* Test that context menu for one of the previously converted widgets like functions widget still works
* Test string widget
    * context menu shows new entries
    * xref, comment, copy address shortcuts work
    * copy string isn't broken
    * filter isn't broken
    * double click seek
    * show in disassembly or hexdump
* flags widget
     * context menu shows new entries
    * rename and delete isn't broken
    * xref, comment, copy address shortcuts work
    * filter and flagspace isn't broken
    * double click seek
    * show in disassembly or hexdump
* search widget
   * new menu entries
   * copy address
   * search isn't broken
   * xrefs shortcut
* edit comment
    * add a comment in any of lists
   * repeat and observed that edit comment dialog opens
* xrefs
   * code preview not broken
   * clicking on xref changes preview  not broken
   * context menu contains new entries
   * opening xref from xref dialog closes the old one 

**Closing issues**

### CODE REVIEW

File: src/widgets/StringsWidget.ui [hunk 1]
```diff
@@ -71,21 +71,11 @@
...
    </layout>
   </widget>
-  <action name="actionCopy_Address">
-   <property name="text">
-    <string>Copy Address</string>
-   </property>
-  </action>
   <action name="actionCopy_String">
    <property name="text">
     <string>Copy String</string>
```
[the remark this answers is not in the document]
Comment (karliss) [reply src/widgets/StringsWidget.ui]: Mixing menu order is little bit annoying but I can do it if you want.

**Comment by karliss** [comment]
Do we need a context menu entry for Ctrl+F if the search bar at the bottom can't be closed?

File: src/widgets/StringsWidget.ui (hunk 1 above)
[the remark this answers is not in the document]
Comment (ITAYC0HEN) [reply src/widgets/StringsWidget.ui]: If it is too much, we can keep it that way

**Comment by ITAYC0HEN** [comment]
> Do we need a context menu entry for Ctrl+F if the search bar at the bottom can't be closed?

Where, for example?
Because I don't find it necessary

**Comment by karliss** [comment]
See your screenshot. It's in the string widget.

**Comment by ITAYC0HEN** [comment]
AS long as the keyboard shortcut of Ctrl+F stays :)

**Pull request merged**